app/views/map/index.html.erb

```diff
@@ -6,58 +6,246 @@
-        <div class = "petpointlogo">
-          <ul>
-            <li>
+    <div id = "right-panel">
+      <h1>Your PetPointer Score!</h1>
+      <div class = "petpointlogo">
+        <section>
+          <aside>
               <img src = "assets/petlogo.png" />
-            </li>
-            <li>
-              <p>
-                Toco Gives This Location
-              </p>
-              <p>
-                A Petscore Of:
-              </p>
-              <p>
-                8 (out of 10)
-              </p>
-            </li>
-          </ul>
-        </div>
+          </aside>
+          <article>
+            <p>
+              Toco Gives This Location
+            </p>
+            <p>
+              A Petscore Of:
+            </p>
+            <p>
+              8 (out of 10)
+            </p>
+          </article>
+        </section>
+      </div>
 
     <p>
       Here Are More Details About Your Pet Score!
     </p>
 
+
       <div class="search-results-list" id="search-results-list">
-        <ul>
-          <li class = "search-result">
-            Parks
-          </li>
-          <li class = "search-result">
-            Vets
-          </li>
-          <li class = "search-result">
-            PetServices
-          </li>
-          <li class = 'search-result'>
-            Lot Size Square Footage
-          </li>
-          <li class = 'search-result'>
-            Interior Square Footage
-          </li>
-          <li class = 'search-result'>
-            Other
-          </li>
-        </ul>
+        <section>
+          <aside>
+            <div class="expander">
+              <a href="javascript:void(0)" class="expander-trigger expander-hidden">Parks(1st One)</a>
+              <div class="expander-content">
+                <ul>
+                  <li>
+                    List of Stores
+                  </li>
+                  <li>
+                    Grocery
+                  </li>
+                  <li>
+                    Eggs
+                  </li>
+                  <li>
+                    FrozenIpsum
+                  </li>
+                  <li>
+                    List of Stores
+                  </li>
+                  <li>
+                    Grocery
+                  </li>
+                  <li>
+                    Eggs
+                  </li>
+                  <li>
+                    FrozenIpsum
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </aside>
+          <article>
+            <p>
+              Parks Score
+            </p>
+            <p>
+              8
+            </p>
+          </article>
+        </section>
+
+        <section>
+          <aside>
+            <div class="expander">
+              <a href="javascript:void(0)" class="expander-trigger expander-hidden">Vets(1st One)</a>
+              <div class="expander-content">
+                <ul>
+                  <li>
+                    List of Stores
```

Comment:
It doesn't matter right now, but short `<li>` elements like this are typically on one line:

```html
<li>List of Stores</li>
```